modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -45,15 +45,16 @@ $ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal --regi
 +
 [source,terminal]
 ----
-$ gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
+$ ingress_backendservice=$(gcloud compute backend-services list --filter="backends.group~${INFRA_ID}" --format='value(name)' | grep -v "${INFRA_ID}")
 ----
+** If `ingress_backendservice` is not empty, run the following commands for the bootstrap group:
```

Comment:
Suggest to replace "run the following commands for the bootstrap group" with "run the following `describe` command, and if it tells the bootstrap group is one of its backends, run `remove-backend` command to remove the bootstrap group from the backends".